packages/router-plugin/tests/router.plugin.spec.ts

```diff
@@ -231,304 +168,6 @@ describe('NgxsRouterPlugin', () => {
     const state = TestBed.get(Store).selectSnapshot(TestState);
     expect(state).toBeTruthy();
     expect(state.url).toEqual('/testpath');
-  }));
-
-  describe('RouterDataResolved', () => {
```

Comment:
Shouldn't we restore this fixture? Or has it moved?

Reply:
This was moved to another file.

Reply:
Ah, thanks! Did any tests disappear from this fixture?